miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/storage/MiniAppSecureStorage.kt

```diff
@@ -59,17 +59,26 @@ internal class MiniAppSecureStorage(
         return status
     }
 
-    private fun validatePreCheck(onFailed: (MiniAppSecureStorageError) -> Unit): Boolean {
-        val status: Boolean = if (!miniAppSecureDatabase.isDatabaseAvailable(databaseName)) {
+    private fun isPreCheckPasses(onFailed: (MiniAppSecureStorageError) -> Unit): Boolean {
```

Comment:
```suggestion
    private fun isPreCheckPassed(onFailed: (MiniAppSecureStorageError) -> Unit): Boolean {
```

Reply:
we are not judging grammar here @shohiebsense 🤣 , the `is` is a convention to add as a prefix to the boolean method `preCheckPasses`. But honestly I'd rather keep the original name. if you want to go this way, you can use `isPrecheckValid` or `hasPrecheckPassed`

Reply:
haha was not being grammar police at all sorry! just a bit strange, agree, it's the preference after all, hence the suggestion, thanks for the thoughts!